db/app.js

```diff
@@ -1,10 +1,15 @@
 const express = require("express");
-const { getTopics, getAPI } = require("../db/controllers");
+const { getTopics, getAPI, getArticleByID } = require("../db/controllers");
 const app = express();
 app.use(express.json());
 
 app.get("/api/topics", getTopics);
 
 app.get("/api", getAPI);
 
+app.get("/api/articles/:article_id", getArticleByID);
+app.use((err, req, res, next) => {
+  console.log(err);
```

Comment:
Careful with leaving console logs in error handlers, as you develop more tests this log will fire many times when running the test suite